README.md

```diff
@@ -1 +1 @@
-# morse-code
+# morse-code

```

Comment:
- [ ] Make sure that you documented your work in a [professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Kindly add a more descriptive readme

Reply:
This issue has been well addressed. Nice job! :clap: